services/pom.xml

```diff
@@ -201,6 +197,11 @@
             <groupId>com.github.ua-parser</groupId>
             <artifactId>uap-java</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.microprofile.openapi</groupId>
+            <artifactId>microprofile-openapi-api</artifactId>
+            <version>3.1</version>
```

Comment:
Minor: the version should be probably specified in the `dependencies-management` in the project root `pom.xml` file. Then it can be ommited here. That's the convention, which we have in Keycloak AFAIK and it helps to avoid different Keycloak components using different versions on various places (Sometimes it is needed, but usually it is desired to use the same version everywhere).

Same likely applies for the plugin versions too (The `pluginManagement` section in the parent pom.xml)

Reply:
I've updated the dependency management config in the `pom.xml`, `rest/admin-ui-ext/pom.xml` and `services/pom.xml` so they should all be in sync now.